src/CMakeLists.txt

```diff
@@ -63,6 +63,9 @@ endif()
 
 function (cxx_add_warning_flags target)
   target_add_compile_flags_if_supported(${target} PRIVATE -Wall -Wextra)
+  target_add_compile_flags_if_supported(${target} PRIVATE
+    -Wcast-qual
+    -Wmissing-field-initializers)
```

Comment:
This option is enabled by `-Wextra` I think. Can you check if its true for the version of `gcc` we're using?

Reply:
Oh shoot, it is.

Reply:
Nice catch!

Reply:
Removed